extensions/amp-story/0.1/amp-story.css

```diff
@@ -869,38 +869,47 @@ amp-story[standalone] .i-amphtml-story-developer-log {
   z-index: 1 !important;
 }
 
-/* TODO(alanorozco): Toast styling for desktop */
 .i-amphtml-story-toast {
   position: fixed !important;
   bottom: 0 !important;
   left: 0 !important;
   right: 0 !important;
   display: inline-block !important;
-  padding: 14px 16px !important;
-  line-height: 16px !important;
+  padding: 1.16em 1.33em !important;
+  line-height: 1.33 !important;
   color: #fff !important;
   background: #212121 !important;
   animation: toast 2.2s !important;
   animation-fill-mode: both !important;
   font-family: 'Roboto', sans-serif !important;
   font-weight: 400 !important;
   font-size: 12px !important;
+  max-width: 640px !important;
+}
+
+@media (min-width: 640px) {
```

Comment:
can we use `desktop` class? so that we are uniform everywhere even if we choose our criteria for desktop at some later point

Reply:
Not without restructuring the `Toast` class, since it appends the element directly onto the body element.

The breakpoint is also different because of the `max-width`.